Tools/scripts/deepfreeze.py

```diff
@@ -326,6 +329,10 @@ def _generate_int_for_bits(self, name: str, i: int, digit: int) -> None:
     def generate_int(self, name: str, i: int) -> str:
         if -5 <= i <= 256:
             return f"(PyObject *)&_PyLong_SMALL_INTS[_PY_NSMALLNEGINTS + {i}]"
+        if i > 0:
+            name = f"const_int_{i}"
+        else:
+            name = f"const_int_negative_{abs(i)}"
```

Comment:
Will not it generate `const_int_negative_0`?

Reply:
``-5 <= i <= 256`` uses cached integers so it is unreachable

Reply:
Nevertheless `>= 0` reads better and avoids the question in the first place.
```suggestion
        if i >= 0:
            name = f"const_int_{i}"
        else:
            name = f"const_int_negative_{abs(i)}"
```